src/start_server.ts

```diff
@@ -16,6 +16,7 @@ import * as assert from 'assert';
 import * as express from 'express';
 import * as fs from 'mz/fs';
 import * as path from 'path';
+import {urlFromPath} from 'polymer-build';
```

Comment:
It is a little misplaced, but I don't really have a better idea at the moment. We don't have a library of utilities like this. Maybe just don't export out of the main polymer-build module, but import directly from `polymer-build/path-transformers`